### PR TITLE
New post for astro view transitions

### PR DESCRIPTION
**Pull request by kaslee**
New file and author entries added

Changes proposed in this pull request:

- New post for astro view transitions
- Author entries added

### CODE REVIEW

**Comment by addyosmani** [comment]
Reviewed internally by @rachelandrew (with suggested fixes merged in this version and a note to be mindful of usage of "we").

**Review by addyosmani — APPROVED** [review]
We did an offline review with fixes landing in a separate commit. LGTM

File: site/en/blog/astro-view-transitions/index.md
Comment (rachelandrew) [inline site/en/blog/astro-view-transitions/index.md:5]:
```suggestion
  Highlight how the Astro community embraced View Transitions when Chrome was shaping up the API and the journey towards landing first-class support for it in Astro + persistent islands.
```

File: site/en/blog/astro-view-transitions/index.md
Comment (rachelandrew) [inline site/en/blog/astro-view-transitions/index.md:77]: all the code is indented, can you remove the white space at the beginning of lines?

File: site/en/blog/astro-view-transitions/index.md
Comment (rachelandrew) [inline site/en/blog/astro-view-transitions/index.md:86]:
```suggestion
Thankfully, the Astro community showed that the View Transitions API was already powerful without the product over-complicating things. After seeing some early demos, the Astro team were sold on the idea of a simple Astro API that could match the browser APIs as closely as possible. Instead of seeing View Transitions as an invisible implementation detail hidden inside of Astro, they could be directly exposed to the developer. Aligning Astro directly with the web platform and reducing overall complexity on the frontend.
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Please see my note about whitespace in code examples.

File: site/en/blog/astro-view-transitions/index.md
Comment (addyosmani) [reply site/en/blog/astro-view-transitions/index.md:77]: Ack. we will do this shortly.

Comment (kaslee) [reply site/en/blog/astro-view-transitions/index.md:77]: Can you check now.

**Review by addyosmani — APPROVED** [review]

**Review by rachelandrew — APPROVED** [review]
Thank you, you should be able to merge this when ready but if you have trouble let me know.

**Review by devnook — APPROVED** [review]

**Comment by kaslee** [comment]
@rachelandrew The latest commit addresses feedback from the Astro team.

Also, I do not have write access to the repo and cannot merge. Request your help for this.

**Pull request merged**
